rubygem-deacon/rubygem-deacon.spec

```diff
@@ -0,0 +1,82 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name deacon
+
+Summary: Human readable names generator
+Name: %{?scl_prefix}rubygem-%{gem_name}
+
+Version: 1.0.0
+Release: 1%{?dist}
+Group: Development/Libraries
+License: GPLv3 and Public Domain
+URL: https://github.com/theforeman/deacon
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+Requires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: %{?scl_prefix_ruby}rubygems
+BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(deacon) = %{version}
+%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
+
+%description
+Deacon ruby library provides human readable names using random (LFSR) generator
+as well as MAC-based algorithm.
+
+%package doc
+BuildArch:  noarch
+Requires:   %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}-doc}
+Summary:    Documentation for rubygem-%{gem_name}
+
+%description doc
+This package contains documentation for rubygem-%{gem_name}.
+
+%prep
+%{?scl:scl enable %{scl} "}
+gem unpack %{SOURCE0}
+%{?scl:"}
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} "}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:"}
+
+%build
+mkdir -p .%{gem_dir}
+
+%{?scl:scl enable %{scl} "}
+# Create the gem as gem install only works on a gem file
+%{?scl:"}
+%{?scl:scl enable %{scl} "}
+gem build %{gem_name}.gemspec
+%{?scl:"}
+
+%{?scl:scl enable %{scl} "}
```

Comment:
If using gem_install, you'll probably also need to use `<<EOF` and `EOF` in this scl wrapper.